content/tutorials/define-and-provision-resources/index.md

```diff
@@ -1,56 +1,64 @@
 ---
-title_tag: "Defining and Provisioning Resources | Tutorials"
-meta_desc: Learn what a resource is, how resources work within Pulumi, and how to create, update, and delete resources in this tutorial.
 title: "Defining and Provisioning Resources"
-h1: "Defining and Provisioning Resources"
-meta_image: /images/docs/meta-images/docs-meta.png
-menu:
-  usingpulumi:
-    identifier: define-provision-resources
-    weight: 1
-tags:
-    - resources
-    - aws
-    - python
-    - nginx
-    - fundamentals
-    - tutorial
+title_tag: "Defining and Provisioning Resources"
+layout: single
+
+# A succinct description of the tutorial. It appears on the Tutorials home and collection pages.
+description: Learn how to define and provision resources in Pulumi.
+
+# A similar description used for search results and social-media previews.
+meta_desc: Learn how to define and provision resources in Pulumi.
+
+# An image for the tutorial. It appears on tutorial page and in social-media previews.
+meta_image: meta.png
+
+# An optional video for the tutorial. When present, it appears at the top of the page, replacing
+# the meta image. YouTube and HTML5 video sources are supported.
+video:
+    youtube: 1UEkiXqxgB8
+
+# The order in which the tutorial appears in most lists. Order is ascending, so higher numbers
+# mean the tutorial will appear further down the list. Positive integers only.
+weight: 999
+
+# A brief summary of the tutorial. It appears at the top of the tutorial page. Markdown is fine.
+summary: |
+    In Pulumi, resources represent the fundamental units that make up your infrastructure, such as virtual machines, networks, storage, and databases. A resource is used to define and manage an infrastructure object in your Pulumi configuration. In this tutorial, you will create a simple Nginx web server. You will then refer to documentation in the Pulumi Registry to create a security group resource to make the application publically accessible.
+
+# A list of three to five things the reader will have learned by the end of the tutorial.
+youll_learn:
+    - How to create a new resource
+    - How to update an existing resource
+    - How to reference resource definitions in the Pulumi documentation
+
+# A list of tutorial prerequisites. Markdown is fine. Keep it simple; no need to be exhaustive here.
+prereqs:
+    - A [Pulumi Cloud account](https://app.pulumi.com/signup) and [access token](/docs/pulumi-cloud/accounts/#access-tokens)
+    - The [Pulumi CLI](/docs/install/)
+    - An [Amazon Web Services](https://aws.amazon.com/) account
+    - The [AWS CLI](https://aws.amazon.com/cli/)
+    - Install [NodeJS](/docs/languages-sdks/javascript/) or [Python](/docs/languages-sdks/python/)
```

Comment:
I think I had an issue with starting the line with a bracket. I think `make serve` was complaining if I remember correctly. As long as this doesn't break anything in the actual build I will add this change.

Reply:
Ah, yes, my bad -- the YAML parser apparently doesn't like lines starting with `[` -- maybe because they look like arrays? At any rate, in the future, you can either do this:

```
prereqs:
    - "[Node.js](/docs/languages-sdks/javascript/) or [Python](/docs/languages-sdks/python/) installed"
```

or tweak it with words:

```
prereqs:
    - Either [Node.js](/docs/languages-sdks/javascript/) or [Python](/docs/languages-sdks/python/) installed
```